### ISSUE TITLE
Accessibility Audit: Navigation between menus and tabs difficult for users with screen readers 

### ISSUE THREAD

**Issue by brittnylapierre** [ProjectMirador/mirador#3880]
Description: An accessibility audit of Mirador was performed. It was found that navigation was difficult for users with limited or no sight. 

Supporting Comments:
- LIMITED SIGHT – iPhone 14 Pro/iOS 17.2/VoiceOver/Safari - Difficult to access gallery view 
- DEAF-BLIND – Windows 10/JAWS 2019/Focus 14 braille/Chrome - Unable to locate the gallery despite efforts 
- LIMITED SIGHT – Windows 10/Windows Narrator/EDGE - In gallery mode, image buttons were present, but Narrator couldn't identify the selected image. 
- BLIND – iPhone 14/OS 17.1.2/VoiceOver/Safari - Inability to change to gallery view or move thumbnails to the bottom and right. 
- BLIND – iPhone 14/OS 17.1.2/ VoiceOver/Safari - Menu did not open when selecting Window View and Thumb Nail Display button. 
- LIMITED SIGHT – Windows 10/NVDA 2022.4/ Chrome - When moving thumbnails to the bottom, a confusing table appears with 1 row and 57 columns of buttons below the gallery. 
- LIMITED SIGHT – Windows 10/NVDA 2022.4/ Chrome - Gallery view adds 57 buttons below menu items, but NVDA doesn't announce anything when pressing these buttons. 
- DEAF – BLIND – Ipad Pro 9/iPadOS 17.2/Focus 40 Blue 4th gen/VoiceOver/Safari - Could not find an information area within the viewer. 
- DEAF-BLIND – Windows 11/laptop/JAWS 2020/PacMate 20 braille/Chrome - Difficulty locating the gallery 
- LIMITED SIGHT – Apple Mac Mini M1 2000/MacOS 13.4.1/27” external monitor/Increased contrast /1280 by 72px/zooming - Took a while to find the menu for gallery view. Worked well once located. Appreciated labelled options in the menu. 
- DEAF-BLIND – iPhone 13 Pro Max/iOS 17.2/Focus 14 Blue 4th gen/VoiceOver/Safari - Expresses difficulty in locating information area within the viewer. 
- DEAF-BLIND – Windows 11/laptop/JAWS 2020/PacMate 20 braille/Chrome - Unable to find information area.
